### PR TITLE
[TwigBridge] do not merge label classes into expanded choice labels

### LINKED ISSUE #40005: [TwigBridge] label class incorrectly applied to expanded choice field
**Symfony version(s) affected**: Symfony 4.4, but it seems to be the same on 5.* also.

**Description**  
Seems to be a bug for some of the form templates for choice field with `expanded` and `multiple` settings set to true.

What I want is to add css class to label of choice class but not the choices. I add `'label_attr' => ['class' => 'red']` as I would to any field. And it actualy works if I set `expanded` to false.

It seems all bootstrap templates suffer from same problem but not `div` and `table` templates.

It this a bug in bootstrap templates?

image

```
ChoiceType::class,
[
    'choices' => ['choice must be black' => 'test'],
    'label' => 'label must be red because of css class',
    'label_attr' => ['class' => 'red'],
    'multiple' => true,
    'expanded' => true,
]
```

**How to reproduce**  
```
git clone https://github.com/aurelijusrozenas/sf-form-choice-template-bug.git
cd sf-form-choice-template-bug
composer install
symfony serve
```

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #40005
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Comment by xabbuh** [comment]
CI failure looks related

Status: Needs work

**Comment by xabbuh** [comment]
After having another look I fail to see how the failure is related.

Status: Needs Review

**Comment by xabbuh** [comment]
ping @symfony/mergers

File: src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig [hunk 1]
```diff
@@ -85,9 +85,6 @@
         {%- if required -%}
             {%- set label_attr = label_attr|merge({class: (label_attr.class|default('') ~ ' required')|trim}) -%}
         {%- endif -%}
-        {%- if parent_label_class is defined -%}
-            {%- set label_attr = label_attr|merge({class: (label_attr.class|default('') ~ ' ' ~ parent_label_class)|trim}) -%}
```
Comment (ogizanagi) [inline src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig:89]: The same logic exists in the Foundation template: https://github.com/symfony/symfony/blob/b560ec4a1d11f7b7e78a4bb9ed75b84397222a26/src/Symfony/Bridge/Twig/Resources/views/Form/foundation_5_layout.html.twig#L256-L258

Should we remove it as well?

Comment (xabbuh) [reply src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig:89]: done

**Review by chalasr — APPROVED** [review]

**Review by yceruto — APPROVED** [review]

**Review by ogizanagi — APPROVED** [review]

**Review by welcoMattic — APPROVED** [review]

**Review by derrabus — APPROVED** [review]

**Comment by fancyweb** [comment]
Thank you @xabbuh.

**Pull request merged**